src/pip/_internal/wheel.py

```diff
@@ -990,7 +1023,13 @@ def _build_one_pep517(self, req, tempd, python_tag=None):
             return None
         return os.path.join(tempd, wheel_name)
 
-    def _build_one_legacy(self, req, tempd, python_tag=None):
+    def _build_one_legacy(
+            self,
+            req,  # type: InstallRequirement
+            tempd,  # type: str
+            python_tag=None  # type: Optional[str]
```

Comment:
I would add a trailing comma and dedent 4 here and in the other function signatures.